__test__/cypress/plugins/index.js

```diff
@@ -5,6 +5,20 @@ require("dotenv").load();
 require("babel-register");
 require("babel-polyfill");
 
+if (process.env.DEFAULT_SERVICE !== "fakeservice") {
+  throw Error(
```

Comment:
Even so, can we do conditionals on DEFAULT_SERVICE == 'fakeservice' for the number buying parts within the cypress tasks themselves?